1.8/development/index.md

```diff
@@ -4,85 +4,13 @@ nav_title: Development
 menu_order: 4
 ---
 
-## <a name="universe"></a>Package Repositories
+# <a name="universe"></a> DC/OS Universe
```

Comment:
`s/DC/OS Universe/Mesosphere Universe/g` There isn't a DC/OS Universe, Mesosphere hosts and runs the Universe, it just so happens DC/OS comes pre-configured to point to the Mesosphere Universe.

Think of this as following the pattern of Android. The Android OS itself does not provide an app store, Google provides the Play Store.